circle.yml

```diff
@@ -0,0 +1,18 @@
+dependencies: 
+  cache_directories: 
+    - ~/.cache/pip
+    - ~/wheelhouse
+  post: 
+    - .circle/services.sh
+    - ".circle/compose.sh run wheezy /bin/true"
+    - ".circle/compose.sh run wheezy"
```

Comment:
I doubt about this. Basically the lines such as  
```
    - sudo pip install wheel
    - sudo pip install docker-compose
```
Don't have any importance, and what they do is just cluttering UI of CircleCI. Less commands in `circle.yml` more easy to navigate between its steps.